src/java/main/org/apache/zookeeper/server/command/FourLetterCommands.java

```diff
@@ -151,9 +153,13 @@
      */
     public final static int telnetCloseCmd = 0xfff4fffd;
 
-    public final static HashMap<Integer, String> cmd2String =
+    final static HashMap<Integer, String> cmd2String =
```

Comment:
We just need to make sure this doesn't break any client exposed public field. Looks like it is **not** this case (that is, we can make it a `Map`), but unfortunately I guess not all concrete declarations in ZK codebase can be changed to interfaces without breaking backward compatibility. It is not that we are not going to change it, but probably only at major release version.